ServerCore/Pages/Resources/pd2020/HomePartial.cshtml

```diff
@@ -14,6 +14,21 @@
     <br class="pd2020-emphasis"/>
 
     <div class="pd2020-container-root">
+        <div class="pd2020-container-card">
+            <h3>FULL SCHEDULE (all times PDT)</h3>
+
+            <h4>INFORMATION SESSIONS (optional)</h4>
+            <p>If you have questions about how Puzzleday works, you can attend one of these four information sessions. We will begin with a short presentation, and take Q&amp;A afterwards. <b><a href="https://aka.ms/pdinfosession">The sessions</a></b> (link updated before each one) will take place on <b>July 1<sup>st</sup></b> and <b>July 7<sup>th</sup></b>, at <b>2pm</b> and again at <b>5pm</b>.</p>
+
+            <h4>PREGAME PUZZLES (optional)</h4>
+            <p>These are a great way to get in the spirit of the event and learn how our website works. We hope they are fun and give you an idea of what to expect.</p>
+            <p>We&rsquo;ll release two puzzles each day on <b>July 6<sup>th</sup></b> through <b>July 9<sup>th</sup></b>, at <b>5pm</b>. To get the puzzles, make sure you are on a registered team and go to the &ldquo;Puzzles&rdquo; link at the top of the page.</p>
+
+            <h4>EVENT WEEKEND</h4>
+            <p>On <b>July 11<sup>th</sup></b>, at <b>10am</b>, we will have a short <b><a href="https://aka.ms/pdopening">opening session</a></b> (link updated shortly before) where we will go over rules and theme. After that, your team will have until <b>1:30pm</b> to solve as many puzzles as you can! We will take a snapshot of the leaderboard at 1:30, but you can keep solving these puzzles at any time afterwards (until the second day is over).</p>
+            <p>On <b>July 12<sup>th</sup></b>, at <b>10am</b>, you will jump right into solving all-new puzzles, and solving will end at <b>1pm</b>. Immediately afterwards, we will have a short <b><a href="https://aka.ms/pdclosing">closing session</a></b> (link updated shortly before) where we will announce prizewinners and stuff!</p>
```

Comment:
Same comment about potentially separating the links out so they're easier to find at a glance. "where we will announce prizewinners and stuff" also sounds oddly informal compared to the rest? What about "where we will announce prizewinners and wrap up the event" ?

Reply:
I'm fixing "stuff" but not the rest, because it's going to make the content too long and also put the intro link towards the end of the event.

Reply:
I changed my mind and did it - looks much better!